mlflow/sklearn/__init__.py

```diff
@@ -334,6 +334,7 @@ def save_model(
 @format_docstring(LOG_MODEL_PARAM_DOCS.format(package_name="scikit-learn"))
 def log_model(
     sk_model,
+    artifact_path: Optional[str] = None,
     name: Optional[str] = None,
```

Comment:
Can we move `name` to the end, since it's new?

Reply:
Moved to the end :)